modules/phase_field/src/kernels/ADACInterface.C

```diff
@@ -18,7 +18,12 @@ ADACInterface::validParams()
   params.addClassDescription("Gradient energy Allen-Cahn Kernel");
   params.addParam<MaterialPropertyName>("mob_name", "L", "The mobility used with the kernel");
   params.addParam<MaterialPropertyName>("kappa_name", "kappa_op", "The kappa used with the kernel");
-  params.addCoupledVar("args", "Vector of nonlinear variable arguments this object depends on");
+  params.addDeprecatedCoupledVar("args",
+                                 "Vector of nonlinear variable arguments this object depends on",
+                                 "args is deprecated, use 'coupled_variables' instead");
```

Comment:
Oh man this is what I get for introducing an API with a default argument. This is the API:
```
  /**                                                                                                                 
   * This method adds a deprecated coupled variable name pair.  The parser will look for variable                     
   * name pair in the input file and can return a reference to the storage location                                   
   * for the coupled variable if found. The doc string for the deprecated variable will be                            
   * constructed from the doc string for the new variable. A deprecation message will also be                         
   * automatically generated                                                                                          
   */                                                                                                                 
  void addDeprecatedCoupledVar(const std::string & old_name,                                                          
                               const std::string & new_name,                                                          
                               const std::string & removal_date = "");
```
This PR introduces ~40 wrong uses

Reply:
I will be deprecating this deprecation API

Reply:
sounds good. 
Hope my PR isnt causing too many troubles